src/xb-builder-node.c

```diff
@@ -889,6 +905,8 @@ static void
 xb_builder_node_class_init (XbBuilderNodeClass *klass)
 {
 	GObjectClass *object_class = G_OBJECT_CLASS (klass);
+
+	object_class->dispose = xb_builder_node_dispose;
```

Comment:
hah! I didn't think of that. I assume valgrind is still clean in the self tests.

Reply:
It is!